crates/solidity/outputs/rust/cli/src/main.rs

```diff
@@ -17,6 +17,9 @@ struct ProgramArgs {
 
     #[clap(long)]
     yaml: Option<String>,
+
+    #[clap(long)]
+    sexpr: Option<String>,
```

Comment:
I'm fine with including `lexpr` for now, but I think we will need to revisit all of the public API as part of #171
Not sure if this is common enough to support. I imagine we can adopt one standard output format (like `JSON`) that is used by the most users, and they can easily transform them to other formats when needed; `yq` for `yaml` for example, or `lexpr` here.

Reply:
Adding this for datalog/prolog experimentation. Although I do think a wide range of outputs is a good idea. Maybe JSON and CBOR.

Reply:
I wanted to suggest starting minimal, and only adding more if there is an important use case. For v1, I think JSON would let us hit all our targets:

2) For CLI environments, most support parsing JSON, and `jq` is widely available.
1) For NPM environments, they can parse it natively without any additional packages/external dependencies.
3) For Rust environments, it doesn't really matter, since they get a `serde` serializable type, and they can produce whatever they want. For example, our own internal experimentation can add a `lexpr` wrapper.